source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACOAdaptiveCards.h

```diff
@@ -0,0 +1,16 @@
+//
+//  ACOAdaptiveCards.h
+//  ACOAdaptiveCards
+//
+//  Copyright © 2017 Microsoft. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "ACOParseResult.h"
+
+@interface ACOAdaptiveCards:NSObject
```

Comment:
>ACOAdaptiveCards [](start = 11, length = 16)

Why is this plural? This just represents one card, right?

Reply:
that's correct; will change to card

---
In reply to: [140379869](https://github.com/Microsoft/AdaptiveCards/pull/632#discussion_r140379869) [](ancestors = 140379869)